codersdk/deployment.go

```diff
@@ -3030,6 +3039,43 @@ Write out the current server config as YAML to stdout.`,
 			Hidden:      true, // Hidden because most operators should not need to modify this.
 		},
 		// Push notifications.
+
+		// Workspace Prebuilds Options
+		{
+			Name:        "Reconciliation Interval",
+			Description: "How often to reconcile workspace prebuilds state.",
+			Flag:        "workspace-prebuilds-reconciliation-interval",
+			Env:         "CODER_WORKSPACE_PREBUILDS_RECONCILIATION_INTERVAL",
+			Value:       &c.Prebuilds.ReconciliationInterval,
+			Default:     (time.Second * 15).String(),
+			Group:       &deploymentGroupPrebuilds,
+			YAML:        "reconciliation_interval",
+			Annotations: serpent.Annotations{}.Mark(annotationFormatDuration, "true"),
+		},
```

Comment:
Should this also be hidden?

Reply:
I think this is a tuneable a few operators might want to fiddle with, but my intuition might be off.
15s feels like a reasonable default, but operators might want prebuilds to reconcile faster.

Reply:
My thought was that if it's an experimental feature and not enabled by default, should the flag be visible by default? 

Alternatively, we could explicitly mention here that it requires a specific experimental flag to be enabled.

Reply:
Good point! I wish we had a mechanism to tie configs to experiments.
I'll give this some thought 👍

Reply:
Done in f6277a02d